client/flutter/lib/main.dart

```diff
@@ -29,7 +32,21 @@ void main() async {
   final bool onboardingComplete =
       await UserPreferences().getOnboardingCompleted();
 
-  runApp(MyApp(showOnboarding: !onboardingComplete));
+  // Set `enableInDevMode` to true to see reports while in debug mode
+  // This is only to be used for confirming that reports are being
+  // submitted as expected. It is not intended to be used for everyday
+  // development.
+  Crashlytics.instance.enableInDevMode = false;
```

Comment:
@britannio have you been able to verify that this does in fact send requests to Firebase Crashlytics on a real device?

Reply:
No, I left a message in Slack asking someone to try it out on IOS, apparently, it takes up to 24 hours for crashes to appear in the console so I'm waiting for now. Just added the blocked label.

Reply:
It hadn't been enabled in Staging - I did that about an hour ago

Reply:
Hi folks, Todd from Crashlytics. If you are testing on simulator you must first press stop in Xcode and then relaunch the app directly from the simulator. Then cause the test crash. Then finally relaunch the app either from the simulator or using Xcode play button.

Reply:
Thanks, Todd.
cc @theswerd

Reply:
I'm only testing on Android but I'm still unable to get logs to show up in the console.
I'm downloading the GoogleServices.json from Firebase, swapping it into the `android/app` folder then using `Crashlytics.instance.crash()` to force a crash.

Reply:
very interesting, can you shoot me an email at firebase-migration@google.com. We have a team here (including me!) to help you out :)

Reply:
Cool, just sent an email

Reply:
Ah, turns out that since `Crashlytics.instance.crash()` doesn't cause a fatal crash it doesn't show up in the Firebase console with the default filter. After removing the filter I'm seeing the crashes :) Thanks for the support Todd.

I'll verify that everything works on IOS before unblocking this PR.